packages/xchain-radix/src/client.ts

```diff
@@ -0,0 +1,642 @@
+import {
+  CommittedTransactionInfo,
+  GatewayApiClient,
+  GatewayStatusResponse,
+  PublicKey,
+  StateEntityDetailsRequest,
+  StateEntityDetailsResponse,
+  TransactionCommittedDetailsRequest,
+  TransactionCommittedDetailsResponse,
+  TransactionPreviewRequest,
+  TransactionPreviewResponse,
+} from '@radixdlt/babylon-gateway-api-sdk'
+import {
+  Convert,
+  Curve,
+  LTSRadixEngineToolkit,
+  NetworkId,
+  PrivateKey,
+  RadixEngineToolkit,
+  TransactionBuilder,
+  TransactionManifest,
+  generateRandomNonce,
+} from '@radixdlt/radix-engine-toolkit'
+import {
+  AssetInfo,
+  Balance,
+  BaseXChainClient,
+  FeeType,
+  Fees,
+  Network,
+  PreparedTx,
+  Tx,
+  TxHistoryParams,
+  TxParams,
+  TxType,
+  TxsPage,
+  XChainClientParams,
+  singleFee,
+} from '@xchainjs/xchain-client'
+import { getSeed } from '@xchainjs/xchain-crypto/lib'
+import { Address, Asset, baseAmount } from '@xchainjs/xchain-util'
+import { bech32m } from 'bech32'
+import BIP32Factory, { BIP32Interface } from 'bip32'
+import { derivePath } from 'ed25519-hd-key'
+import * as ecc from 'tiny-secp256k1'
+// eslint-disable-next-line ordered-imports/ordered-imports
+import {
+  MAINNET_GATEWAY_URL,
+  RadixChain,
+  STOKENET_GATEWAY_URL,
+  XRD_DECIMAL,
+  XrdAsset,
+  bech32Networks,
+  xrdRootDerivationPaths,
+} from './const'
+
+/**
+ * Custom Radix client
+ */
+
+export default class Client extends BaseXChainClient {
+  gatewayApiClient: GatewayApiClient
+  curve: Curve
+  constructor(
+    {
+      network = Network.Mainnet,
+      phrase,
+      rootDerivationPaths = xrdRootDerivationPaths,
+      feeBounds = {
```

Comment:
This fee bound is too strict, is this in XRD? If so, can we make it between 0 to 3 XRD?

Reply:
yes